package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "connector-registry-storage-adapter",
+  "version": "1.0.0",
+  "description": "Adapter for Connector Registry github storage",
+  "main": "dist/index.js",
```

Comment:
Why `dist`, not `src`, for example?

Reply:
Because its transpiled typescript code. That is common practice to name folder "dist" for transpiled code. To see examples you can go to node_modules packages and look how it named there. SRC for source code, dist for transpiled code (or for compiled code in languages that must be compiled to use)